src/pages/docs/getting-started/first-deployment/index.mdx

```diff
@@ -1,120 +1,169 @@
+The next step is creating your deployment process. This is where you define the steps that Octopus uses to deploy your software. 
 
-Select the environments you'd like to create and click **SAVE**.
-![Add Environments](/docs/getting-started/first-deployment/images/img-createenvironmentdialog.png "width=500")
+For this deployment, we will configure one step to print _Hello World_.
 
-## Project questionnaire
+1. In the dialog, click **Thanks, got it**.
 
-You have the option to fill out a short survey. This helps our team learn about the technologies our customers are using, which guides the future direction of Octopus. It should only take about 30 seconds to complete.
+Octopus lands you in the process step template library.
 
-Click **SUBMIT** and you will be taken to your project.
+2. In the **Featured** category, locate the Run a Script card and click **Add Step**.
 
 :::figure
-![Project questionnaire](/docs/getting-started/first-deployment/images/img-questionnairedialog.png "width=500")
+![Featured step templates](/docs/getting-started/first-deployment/images/add-step.png)
 :::
 
-## Create deployment process
+### Step Name
+You can leave this as the default _Run a Script_.
+
+### Script Source
+
+You can source script files via 3 methods:
+
+* Inline script (default)
+* Git repository
+* Package
 
-The next step in the journey is to create your deployment process. This is where you define the steps that Octopus uses to deploy your software. For our simple Hello World deployment, we will configure one step to print _Hello World_.
+3. Select **Inline script** as your script source. 
 
 :::figure
-![Create deployment process](/docs/getting-started/first-deployment/images/img-createdeploymentprocess.png "width=500")
+![Script source expander where users can select where to source scripts from](/docs/getting-started/first-deployment/images/script-source.png)
 :::
 
-1. Click **CREATE PROCESS** to see the available deployment steps.
-2. Select the **Script** tile to filter the types of steps.
-3. Scroll down and click **ADD** on the **Run a Script** tile.
-4. Accept the default name for the script.
-5. In the Execution Location section, select **Run once on a worker** if you're using Octopus Cloud, or **Run once on the Octopus Server** if you're using a self-hosted Octopus instance.
-6. Scroll down to the Script section, select your script language of choice, and enter the following script in the Inline Source Code section. 
+### Inline Source Code
+
+4. Select an appropriate script language.
+5. **Copy** the script below and **paste** it into the source code editor.
+
 ```
 Write-Host "Hello, World!"
 ```
 
-:::div{.hint}
-If you are using Octopus Cloud, Bash scripts require you to select the Hosted Ubuntu worker pool. The Default Worker Pool is running Windows and doesn't have Bash installed.
+:::figure
+![Inline source code expander where users can type a script](/docs/getting-started/first-deployment/images/inline-source-code.png)
```

Comment:
This image displays the script editor referencing a variable, which is a different result compared to what we've instructed the user to do (it should just say `Write-Host "Hello, World!"` as per the code snippet above):

<img width="823" alt="image" src="https://github.com/user-attachments/assets/37bf5518-34fd-4165-a354-97c4a8ec5c51">

Reply:
Image fixed in this [commit](https://github.com/OctopusDeploy/docs/pull/2560/commits/d8c309af96c57c3dc877d547d8706274ba75d672)